src/main/groovy/net/vivin/gradle/versioning/SemanticBuildVersion.groovy

```diff
@@ -19,7 +19,7 @@ class SemanticBuildVersion {
 
     String snapshotSuffix = "SNAPSHOT"
 
-    Pattern tagPattern = ~/.*/
+    Pattern tagPattern = ~/\d++\.\d++\.\d++/
```

Comment:
I'm not familiar with this syntax (the `++` part). Couldn't the default pattern be something like `^.*\d+\.\d+\.\d+`?